plugins/main/public/components/common/modules/modules-defaults.tsx

```diff
@@ -111,8 +113,24 @@ export const ModulesDefaults = {
   gcp: {
     init: 'dashboard',
     tabs: [
-      DashboardTab,
-      renderDiscoverTab(DEFAULT_INDEX_PATTERN, googleCloudColumns),
+      {
+        id: 'dashboard',
+        name: 'Dashboard',
+        component: DashboardGoogleCloud,
+        /* For ButtonModuleExploreAgent to insert correctly according to the module's index pattern, the moduleIndexPatternTitle parameter is added. By default it applies the index patternt wazuh-alerts-* */
+        buttons: [
+          ({ ...props }) => (
+            <ButtonModuleExploreAgent
+              {...props}
+              moduleIndexPatternTitle={WAZUH_VULNERABILITIES_PATTERN}
```

Comment:
Why is this being done? Shouldn't it be the normal button like on the other dashboards?